### PR TITLE
Add FYQ2 Roadmap

### PR DESCRIPTION
**Pull request by amanifarooque**

### CODE REVIEW

File: _posts/2021-01-25-roadmap.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+## FY21, Q2: January - March 2021
+
+### Now
+
+* **Version Upgrades**
+  * We will be upgrading our Elasticsearch and Ruby components to keep them up-to-date and compliant. 
+* **Search.gov website redesign**
+  * We will be launching a new design that aligns with the [US Web Design System](https://designsystem.digital.gov/), in accordance with the [21st Century IDEA](https://digital.gov/resources/21st-century-integrated-digital-experience-act/). We are excited for your feedback!
+* **Search.gov Search Results Page (SERP) Redesign**
+  * We will begin the early stages of scoping out the work required to upgrade the SERP and decommission out-of-date variants of the out-of-the-box experience.
```
Comment (dawnpm) [inline _posts/2021-01-25-roadmap.md:31]: @amanifarooque can we lighten this up? Maybe just use the language from the last roadmap?

File: _posts/2021-01-25-roadmap.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+* **Indexing improvements**
+  * We are fixing our indexing on the back-end to support non-standard site structures.
```
Comment (dawnpm) [inline _posts/2021-01-25-roadmap.md:33]: which work is this referencing?

File: _posts/2021-01-25-roadmap.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+* **Continuity of Operations Plan Improvements** 
+  * We are taking steps to bolster our system's resiliance and reliability.
+
+### Next
+
+* **Development process improvements**
+  * Streamline our deployment process
+  * Update our system error diagnostics tooling
+  * Clean out legacy code components
+* **Results page redesign**
```
Comment (dawnpm) [inline _posts/2021-01-25-roadmap.md:43]: Can we put make this the first item in this section?

**Review by dawnpm — COMMENTED** [review]
Hey, thanks. A few questions, see embedded comments.

File: _posts/2021-01-25-roadmap.md (hunk 2 above)
Comment (amanifarooque) [reply _posts/2021-01-25-roadmap.md:33]: Referencing closing out bulk upload - updating this to - "We are improving our indexing workflows on the back-end to support non-standard site structures."

File: _posts/2021-01-25-roadmap.md (hunk 1 above)
Comment (amanifarooque) [reply _posts/2021-01-25-roadmap.md:31]: Copied the text for the website redesign, and slimmed down the SERP redesign bullet!

File: _posts/2021-01-25-roadmap.md (hunk 3 above)
Comment (amanifarooque) [reply _posts/2021-01-25-roadmap.md:43]: Done!

**Pull request merged**